nat/src/stateful/mod.rs

```diff
@@ -473,14 +407,14 @@ impl StatefulNat {
 
         match net {
             Net::Ipv4(_) => {
-                let tuple = Self::extract_tuple(packet, src_vpc_id, dst_vpc_id)
-                    .ok_or(StatefulNatError::TupleParseError)?;
-                self.translate_packet::<Buf, Ipv4Addr>(packet, &tuple, src_vpc_id, dst_vpc_id)
+                let flow_key =
+                    Self::extract_flow_key(packet).ok_or(StatefulNatError::TupleParseError)?;
```

Comment:
There is a TryFrom method for doing this extraction, do we want to delete the `extract_flow_key` method now?